### ISSUE TITLE
get smoothed labels to work with generator

### ISSUE THREAD

**Issue by AnthonyRentsch** [AnthonyRentsch/AC297r_2019_somerville_parking#29]

**Comment by AnthonyRentsch** [comment]
Might have to apply smoothing in the loss function.

See: https://towardsdatascience.com/dcgans-generating-dog-images-with-tensorflow-and-keras-fb51a1071432

**Comment by AnthonyRentsch** [comment]
- [x] Edit function to smooth labels
- [x] Write new loss function
- [ ] Add new loss function to model
